src/components/Person/Person.tsx

```diff
@@ -0,0 +1,55 @@
+import { Person as IPerson } from '../../types';
+import React from 'react';
+import cn from 'classnames';
+import { NavLink, useParams } from 'react-router-dom';
+
+interface Props {
+  person: IPerson;
+}
+
+export const Person: React.FC<Props> = ({ person }) => {
+  const { personSlug } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': personSlug === person.slug })}
+    >
+      <td>
+        <NavLink
+          to={`/people/${person.slug}`}
+          className={cn({ 'has-text-danger': person.sex === 'f' })}
+        >
+          {person.name}
+        </NavLink>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+
+      {person.mother ? (
+        <td>
+          <NavLink
```

Comment:
You can use `<Link>`. The NavLink  is used for creating navigation links that can automatically apply styles or classes to the active link based on the current route.